src/length.js

```diff
@@ -1,5 +1,5 @@
 var _curry1 = require('./internal/_curry1');
-var is = require('./is');
+var _isNumber = require('./internal/_isNumber');
```

Comment:
[`S.is`][1] does not rely on `instanceof`. If we're okay with losing support for inheritance (I am) we can use the same approach. If we don't want to support `@@type` the implementation is straightforward:

```javascript
function is(ctor, x) {
  var match = String(ctor).match(/function (\w*)/);
  return match != null && match[1] === R.type(x);
}
```


[1]: http://sanctuary.js.org/#is